### PR TITLE
REFACTOR: Style to home autenticated

### LINKED ISSUE #34: Style home registered view
Apply styles to the home view if the user is authenticated

### PR DESCRIPTION
**Pull request by brayandiazc**
# Description 
Refactoring and correcting views

Github issue: #34

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
-    <p>
-      <%= complaint.category.name %> - <%= complaint.date_of_events %> </br>
-      <%= complaint.state.name %> <%= complaint.municipality.name %> <%= complaint.postal.name %> <%= complaint.settlement.name %> <%= complaint.street %> <%= complaint.number %></br>
-      <strong><%= complaint.title %></strong> </br>
-      <%= complaint.description %> </br>
-    </p>
-  <% end %>
-</div>
\ No newline at end of file
+<div class="columns">
```
Comment (RobertVel666) [inline app/views/home/index.html.erb:1]: Did not know off this framework, great use!

**Review by Brayan9105 — APPROVED** [review]
Bulma make really esay the front end build components

**Comment by brayandiazc** [comment]
Yes, that's why I like it so much

**Review by RobertVel666 — APPROVED** [review]
nice use of bulma columns!

**Pull request merged**